runtime/testy/runtime.py

```diff
@@ -86,7 +86,7 @@ def nonTestModePrint(*args):
           spawnProcess(PROCESS_NAMES.UDP_RECEIVE_PROCESS, startUDPReceiver)
           dawn_connected = False
           controlState = "idle"
-          break
+          continue
```

Comment:
Yes, we should break since if dawn actually disconnects, we should probably stop the robot from moving. In terms of potential issues with shoddy connections, the TCP socket which we use to detect disconnects should have its own timeouts.